covid19_sfbayarea/data/sonoma.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/env python3
+import requests
+import json
+import re
+import dateutil.parser
+from typing import List, Dict, Union
+from bs4 import BeautifulSoup, element # type: ignore
+from format_error import FormatError # type: ignore
```

Comment:
This is broken for me. If I run `./run_scraper_data.sh solano` or `python scraper_data.py solano`, I get:

```
Traceback (most recent call last):
  File "scraper_data.py", line 4, in <module>
    from covid19_sfbayarea import data as data_scrapers
  File "/Users/rbrackett/Dev/sfbrigade/data-covid19-sfbayarea/covid19_sfbayarea/data/__init__.py", line 4, in <module>
    from . import sonoma
  File "/Users/rbrackett/Dev/sfbrigade/data-covid19-sfbayarea/covid19_sfbayarea/data/sonoma.py", line 8, in <module>
    from format_error import FormatError
ModuleNotFoundError: No module named 'format_error'
```

Does it work for you? Seems like it should be:

```suggestion
from .format_error import FormatError
```

(It also shouldn’t need need the `# type: ignore` comment.)

---

Also, while you’re here, maybe it’s worthwhile to make this a shared module up in `covid19_sfbayarea/errors.py` instead of `covid19_sfbayarea/data/format_error.py`? Some of the news scrapers also use [an identical class](https://github.com/sfbrigade/data-covid19-sfbayarea/blob/47ced984c3379fa4b5908e3de1e9778714d5e7c4/covid19_sfbayarea/news/errors.py#L1-L7), and we should really only have one implementation. Then this line would be:

```suggestion
from ..errors import FormatError
```